[2025-01-11T08:48:50.006+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_teams_data.send_teams_to_kafka manual__2025-01-11T08:48:47.283611+00:00 [queued]>
[2025-01-11T08:48:50.011+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_teams_data.send_teams_to_kafka manual__2025-01-11T08:48:47.283611+00:00 [queued]>
[2025-01-11T08:48:50.011+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-01-11T08:48:50.018+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): send_teams_to_kafka> on 2025-01-11 08:48:47.283611+00:00
[2025-01-11T08:48:50.022+0000] {standard_task_runner.py:57} INFO - Started process 153 to run task
[2025-01-11T08:48:50.025+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fetch_teams_data', 'send_teams_to_kafka', 'manual__2025-01-11T08:48:47.283611+00:00', '--job-id', '768', '--raw', '--subdir', 'DAGS_FOLDER/process_teams_dag.py', '--cfg-path', '/tmp/tmph3hy_rs_']
[2025-01-11T08:48:50.027+0000] {standard_task_runner.py:85} INFO - Job 768: Subtask send_teams_to_kafka
[2025-01-11T08:48:50.060+0000] {task_command.py:410} INFO - Running <TaskInstance: fetch_teams_data.send_teams_to_kafka manual__2025-01-11T08:48:47.283611+00:00 [running]> on host fad25223cfa4
[2025-01-11T08:48:50.118+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_teams_data' AIRFLOW_CTX_TASK_ID='send_teams_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-01-11T08:48:47.283611+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-11T08:48:47.283611+00:00'
[2025-01-11T08:48:50.194+0000] {logging_mixin.py:150} INFO - All team data sent to 'teams' topic in Kafka.
[2025-01-11T08:48:50.195+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-01-11T08:48:50.208+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=fetch_teams_data, task_id=send_teams_to_kafka, execution_date=20250111T084847, start_date=20250111T084850, end_date=20250111T084850
[2025-01-11T08:48:50.248+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-01-11T08:48:50.265+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
